packages/rum/README.md

```diff
@@ -272,17 +275,17 @@ datadogRum.init({
 });
 ```
 
-For example, to collect 100% of your sessions using only the Session Replay RUM option without recording a replay:
+For example, to collect 100% of your sessions with Session Replay RUM data:
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** for symmetry sake‏
```suggestion
For example, to collect 100% of your sessions with only Session Replay RUM data:
```

Reply:
I think this is kind of confusing, because Session Replay RUM data includes Browser RUM data. Telling "only" would indicate that it's not the case. For example, a bartender wouldn't say "It's 2€ for Coke only and 5€ for RUM & Coke only" :)

Reply:
We are saying that we can have either x or y and then showing option to collect only x or only y.
I am wondering if it not adding "data" that makes things confusing, since we are now talking about content rather than type.
Maybe talking about Browser RUM sessions and Session Replay RUM sessions could be clearer.
wdyt?

Reply:
WDYT of https://gist.github.com/BenoitZugmeyer/7f6902897a45bd8879c93a92107f685f ?